Calling eprintf in read_uleb128 is causing a format-security build warning

```
home/phakeobj/code/radare2/libr/..//libr/bin/p/../format/mach0/mach0.c: In function ‘read_uleb128’:
/home/phakeobj/code/radare2/libr/..//libr/bin/p/../format/mach0/mach0.c:38:3: warning: format not a string literal and no format arguments [-Wformat-security]
   eprintf (error);
   ^~~~~~~

...

In file included from /home/phakeobj/code/radare2/libr/..//libr/bin/p/../format/mach0/mach064.c:4:0:
/home/phakeobj/code/radare2/libr/..//libr/bin/p/../format/mach0/mach0.c: In function ‘read_uleb128’:
/home/phakeobj/code/radare2/libr/..//libr/bin/p/../format/mach0/mach0.c:38:3: warning: format not a string literal and no format arguments [-Wformat-security]
   eprintf (error);
   ^~~~~~~
```

--- a/libr/bin/format/mach0/mach0.c
+++ b/libr/bin/format/mach0/mach0.c
@@ -35,7 +35,7 @@ static ut64 read_uleb128(ut8 **p, ut8 *end) {
 	ut64 v;
 	*p = (ut8 *)r_uleb128 (*p, end - *p, &v, &error);
 	if (error) {
-		eprintf (error);
+		eprintf ("%s", error);
 		R_FREE (error);
 		return UT64_MAX;
 	}

